mithril-explorer/components/Artifacts/MithrilStakeDistributionsList/index.js

```diff
@@ -62,6 +62,7 @@ export default function MithrilStakeDistributionsList(props) {
                       <Card.Title>{mithrilStakeDistribution.hash}</Card.Title>
                       <ListGroup variant="flush" className="data-list-group">
                         <ListGroup.Item>Epoch: {mithrilStakeDistribution.epoch}</ListGroup.Item>
+                        <ListGroup.Item>Created: {mithrilStakeDistribution.created_at}</ListGroup.Item>
```

Comment:
Maybe we can add this in a separate PR if this is a breaking change for the explorer?
This may break the `pre-release` and `release` environments until they run this PR.

Reply:
Alternatively you can show this field only if it exist with something like this:
```js
{mithrilStakeDistribution.created_at &&
  <ListGroup.Item>Created: {mithrilStakeDistribution.created_at}</ListGroup.Item>
}
```

Reply:
This commit has been isolated into a new branch so we can deploy the explorer on its own.